pkg/networkservice/acl/common.go

```diff
@@ -98,8 +98,15 @@ func aclAdd(tag string, egress bool, aRules []acl_types.ACLRule) *acl.ACLAddRepl
 		R:        aRules,
 	}
 	if egress {
-		for _, a := range aclAddReplace.R {
-			a.SrcPrefix, a.DstPrefix = a.DstPrefix, a.SrcPrefix
+		for i := range aclAddReplace.R {
+			aclAddReplace.R[i].SrcPrefix, aclAddReplace.R[i].DstPrefix =
+				aclAddReplace.R[i].DstPrefix, aclAddReplace.R[i].SrcPrefix
+
+			aclAddReplace.R[i].SrcportOrIcmptypeFirst, aclAddReplace.R[i].DstportOrIcmpcodeFirst =
+				aclAddReplace.R[i].DstportOrIcmpcodeFirst, aclAddReplace.R[i].SrcportOrIcmptypeFirst
+
+			aclAddReplace.R[i].SrcportOrIcmptypeLast, aclAddReplace.R[i].DstportOrIcmpcodeLast =
+				aclAddReplace.R[i].DstportOrIcmpcodeLast, aclAddReplace.R[i].SrcportOrIcmptypeLast
```

Comment:
As I can see this is not related to this PR.

Can you please provide this fix in separate PR?

Reply:
ok, done
https://github.com/networkservicemesh/sdk-vpp/pull/293